src/components/AttendeeList.tsx

```diff
@@ -0,0 +1,47 @@
+import { AttendeeListProps } from "@/interfaces/propsInterfaces";
+import React from "react";
+
+function AttendeeList({ attendees }: AttendeeListProps) {
+  return (
+    <div className="relative overflow-x-auto shadow-md sm:rounded-lg">
+      <table className="min-w-full text-sm text-left text-gray-500 dark:text-gray-400">
+        <thead className="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
+          <tr>
+            <th scope="col" className="px-6 py-3">
+              Name
+            </th>
+            <th scope="col" className="px-6 py-3">
+              Email
+            </th>
+            <th scope="col" className="px-6 py-3">
+              Phone
+            </th>
+            <th scope="col" className="px-6 py-3">
+              Gender
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {attendees.map((attendee, index) => (
+            <tr
+              key={index}
```

Comment:
Are you sure that the `attendees` array would not change? It is not recommended to pass the index as a `key` in React. Please check [this React doc](https://react.dev/learn/rendering-lists#why-does-react-need-keys).

Check if the `attendees` array items have a unique value that can be used as `key` for each item instead of the index.

Reply:
<img width="598" alt="image" src="https://github.com/user-attachments/assets/42a2df16-f08f-41fb-9503-c957e904a328">


the only unique identifier here in the attendees sub collection is email